test/testdrive/mzcompose.yml

```diff
@@ -8,29 +8,95 @@
 # by the Apache License, Version 2.0.
 
 version: '3.7'
+
+mzworkflows:
+  # Run tests with no support for s3
+  local:
+    steps:
+      - step: workflow
+        workflow: start-deps
+      - step: run
+        service: testdrive
+
+  # Run tests with s3 stubbed out, no credentials needed
+  local-aws:
+    env:
+      AWS_REGION: localstack
+      AWS_ENDPOINT: http://localstack:4566
+    steps:
+      - step: start-services
+        services: [localstack]
+      - step: workflow
+        workflow: start-deps
+      - step: wait-for-tcp
+        host: localstack
+        port: 4566
+      - step: run
+        service: testdrive
+
+  # Run tests, requires credentials to be present. See guide-testing for
+  # details.
+  remote:
+    env:
+      AWS_REGION: us-east-2
+      AWS_ENDPOINT: ''
+    steps:
+      - step: workflow
+        workflow: start-deps
+      - step: run
+        service: testdrive
+        command: ${TD_TEST:-*.td esoteric/*.td}
```

Comment:
This isn't different from the `ci` test as far as I can tell. Let's just nuke this one? If folks want to do the real thing I think typing `./mzcompose run ci` is fine!